lldb/source/Symbol/LineEntry.cpp

```diff
@@ -20,7 +20,7 @@ LineEntry::LineEntry()
 void LineEntry::Clear() {
   range.Clear();
   file.Clear();
-  original_file.Clear();
+  original_file = std::make_shared<SupportFile>();
```

Comment:
Is this default-constructing an empty object? Should it be a null shared_ptr?

Reply:
Correct, this ensures I can always assume that the shared pointer is valid. The underlying FileSpec might be "empty" which matches the existing semantics.